dataset/src/main/kotlin/org/jetbrains/kotlinx/dl/dataset/preprocessor/image/Loading.kt

```diff
@@ -16,7 +16,7 @@ import java.nio.file.Path
 import kotlin.streams.toList
 
 /**
- * This image preprocessor defines the Loading operation.
+ * This class defines the Loading operation.
```

Comment:
Highly likely it should be moved to the first-level in the DSL like

   loadFile {
                    pathToData = File("test.jpg")
                    imageShape = ImageShape(20, 20, 3)
                    colorMode = ColorOrder.BGR
                }
transformImage {
                resize {
                    outputWidth = 100
                    outputHeight = 100
                    interpolation = InterpolationType.NEAREST
                }
            }

And also we could add loadFile/loadDirectories ops or 
load {
      file {
      }
}

load {
      directory {
      }
}

load {
      database {
      }
}

Reply:
But it could be a following ticket of course, it's not a part of this refactoring